### ISSUE TITLE
Docker issue !

### ISSUE THREAD

**Issue by hedho** [SolidInvoice/SolidInvoice#816]
Pull the desired full version of the SolidInvoice image. Replace <VERSION> with the desired version number, e.g., 2.2.2-full:

Doesn't have any information over how to create an account or what is the default account !

**Comment by pierredup** [comment]
@hedho When running the full image and opening the app in your browser, it should then give you a screen to create a new user. If this does not happen, what do you see when running the docker image and opening http://localhost/ for the first time?

**Comment by hedho** [comment]
> @hedho When running the full image and opening the app in your browser, it should then give you a screen to create a new user. If this does not happen, what do you see when running the docker image and opening http://localhost/ for the first time?

I see straight a login form which from github sources I was able to find that the default user is "admin" and password is "password" but either then Even selecting users for creating Invoices says select user even though the client is selected if needed image proof I would reinstall again and upload some images !

Basically I found the login info here: https://raw.githubusercontent.com/SolidInvoice/SolidInvoice/2.3.x/docker/Dockerfile.full-build

Reffering: 

```
        --admin-username=admin \
        --admin-password=password \
        --admin-email=test@example.com && \
```

Here image provided
From the first time running the setup

![image](https://i.imgur.com/S0oU2jj.jpeg)
![image](https://i.imgur.com/EQUhGyx.png)
![image](https://i.imgur.com/e8yRFKZ.png)
![image](https://i.imgur.com/IjfEbDH.png)
![image](https://i.imgur.com/dXY93sC.png)

**Comment by pierredup** [comment]
Thanks for the info.

> Basically I found the login info here: [2.3.x/docker/Dockerfile.full-build (raw)](https://raw.githubusercontent.com/SolidInvoice/SolidInvoice/2.3.x/docker/Dockerfile.full-build)

That was just test data, and shouldn't have been part of the image. I have removed this info, and pushed a new Docker image.

> Even selecting users for creating Invoices says select user even though the client is selected

This is definitely a bug. I will look into this and see if I can get a fix out.

**Comment by hedho** [comment]
Thank you @pierredup for understanding, I hope you can fix it easy and fast :)

**Comment by hedho** [comment]
The bug where it requires for selecting user is present even on https://solidinvoice.app/ o.0
And I was using the stable build aswell on my instance 2.2.2 for full build

![brave_AN1Di7JwG2](https://github.com/SolidInvoice/SolidInvoice/assets/224877/b898c78d-d4b6-471e-809e-9a1604d3641b)

**Comment by pierredup** [comment]
The issue has been fixed in #819 and will be included in the 2.2.3 release.
